demo/kubernetes/rook-storageclass.yaml

```diff
@@ -1,3 +1,18 @@
+apiVersion: rook.io/v1alpha1
+kind: Rookpool
+metadata:
+  name: rook-replicapool
+spec:
+  name: replicapool
```

Comment:
The rookpool is namespace-scoped. So if we have multiple rook clusters in different namespaces, then we can have pools associated to each clusters on that namespace. Unless we have rook clusters in the same namespace which i dont think is correct.